client/question/components/MonospaceDisplayModalDirective.js

```diff
@@ -0,0 +1,129 @@
+// Copyright 2017 The TIE Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Directive for the modal displaying monospaced code/error
+ * messages.
+ */
+
+tie.directive('monospaceDisplayModal', [function() {
+  return {
+    restrict: 'E',
+    scope: {},
+    template: `
+      <div class="container">
+        <header>
+          <div class="table-cell title">
+            {{title}}
+          </div>
+        </header>
+        <div class="body">
+          <div class="table-cell content-container">
+            <div class="content">
+              <div ng-repeat="line in contentLines track by $index">
+                {{line}}
+              </div>
+            </div>
+          </div>
+        </div>
+        <footer>
```

Comment:
Unsure where it's actually coming from (it shows up on the <footer> when inspecting), but there's like lines on the corner that indicate the window is resizeable. If we aren't allowing that, should probably remove it? The one on the feedback works.

Reply:
Oh, good point. I was actually thinking that it's probably OK for the modal window to be resizable but not movable, similar to the feedback window that it replaces. @rabidbit could you confirm?

Reply:
The modal window should not be resizable. It should be static in position and size (completely cover the feedback window and related controls). Allowing it to be resizable makes it more complicated as the feedback window would then either also have to be resized at the same time (which is not intuitive) or the modal could be resized to expose the feedback window behind it (which is what we want to avoid).

Reply:
Sounds good, thanks for the confirmation. This is fixed.